style/servo/media_queries.rs

```diff
@@ -357,8 +362,24 @@ fn eval_device_pixel_ratio(context: &Context) -> f32 {
     eval_resolution(context).dppx()
 }
 
+/// Values for the prefers-color-scheme media feature.
+#[derive(Clone, Copy, Debug, FromPrimitive, Parse, PartialEq, ToCss)]
+#[repr(u8)]
+#[allow(missing_docs)]
+pub enum PrefersColorScheme {
```

Comment:
Emilio prefers sharing these things between Servo and Gecko: https://phabricator.services.mozilla.com/D226265?vs=on&id=932339#inline-1256740

Reply:
I actually have code for that already! (which I didn't push because I thought it might be better to do "one thing at a time"). I've pushed it now. Does that seem better?

Reply:
Yeah, well, no big deal either way if you prefer to leave that for a follow-up